### ISSUE TITLE
`NullPointerException` on `CoordinatorLayout`

### ISSUE THREAD

**Issue by eliasp** [stefan-niedermann/nextcloud-deck#1356]
**Describe the bug**
While I was using another app, a Nextcloud Deck crash dialog popped up.

**Steps to reproduce the behavior:**
I can't reproduce the crash, but maybe the stacktrace helps identifying + fixing what happened.
Feel free to close this issue if this doesn't lead anywhere.

**Expected behavior**
The app shouldn't crash in the background, even though the response from the server might be invalid

**Versions**

 - Nextcloud: 24.0.6
 - Nextcloud Deck: 1.7.1
 - Nextcloud Android: 3.21.2
 - Nextcloud Android Deck: 1.20.3

**Smartphone (please complete the following information):**
- Device: OnePlus Nord (AC2003)
- Android-Version: 11
- App-Store:
  - [x] Google Play Store
  - [ ] Google Play Store (Beta channel)
  - [x] F-Droid
  - [ ] Huawei AppGallery

**Stacktrace**
```
Full Crash:

App Version: 1.20.3
App Version Code: 1020003
App Flavor: fdroid

Files App Version Code: 30210290

---

OS Version: 4.19.113-perf+(2104202034)
OS API Level: 30
Device: Nord
Manufacturer: OnePlus
Model (and Product): AC2003 (Nord_EEA)

---

java.lang.NullPointerException: Attempt to read from field 'androidx.coordinatorlayout.widget.CoordinatorLayout it.niedermann.nextcloud.deck.databinding.ActivityMainBinding.coordinatorLayout' on a null object reference
	at it.niedermann.nextcloud.deck.ui.MainActivity.lambda$showSyncFailedSnackbar$34$it-niedermann-nextcloud-deck-ui-MainActivity(MainActivity.java:1115)
	at it.niedermann.nextcloud.deck.ui.MainActivity$$ExternalSyntheticLambda9.run(Unknown Source:4)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:245)
	at android.app.ActivityThread.main(ActivityThread.java:8004)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:631)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:978)

```

**Comment by stefan-niedermann** [comment]
Looks like the same issue as in https://github.com/stefan-niedermann/nextcloud-deck/issues/1351

I already had a quick look at it and it seems that when the response of a UI triggered synchronization arrives and the UI has been destroyed in the meantime (e.g. by closing the app), this can happen.

I will firther investigate the issue, it should be fixable. Thanks for the report.

**Comment by stefan-niedermann** [comment]
Will be fixed in `1.21.0`. Please reopen or create a new issue in case this happens with `1.21.0` or later. Thank you for the report.
